CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Include/exclude recommendations ([#5](https://github.com/stac-api-extensions/fields/pull/5))
```

Comment:
this should be more clear, like "Include/exclude semantics more precisely defined" -- they're not just recommendations.